sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewSuite.scala

```diff
@@ -159,7 +159,9 @@ abstract class SQLViewSuite extends QueryTest with SQLTestUtils {
         Thread.currentThread().getContextClassLoader.getResource("data/files/employee.dat")
       assertNoSuchTable(s"""LOAD DATA LOCAL INPATH "$dataFilePath" INTO TABLE $viewName""")
       assertNoSuchTable(s"TRUNCATE TABLE $viewName")
-      assertNoSuchTable(s"SHOW CREATE TABLE $viewName")
+      intercept[AnalysisException] {
```

Comment:
As you know already, we need to check the actual error message always because `AnalysisException` can hide the regressions.

Reply:
Fixed